common.sh

```diff
@@ -71,7 +71,7 @@ create_pg()
     if [ ${ENABLE_PLACEMENT_GROUP} -eq 0 ]; then
         return 0
     fi
-    PLACEMENT_GROUP="slave-pg-$RANDOM"
+    PLACEMENT_GROUP="slave-pg-$RANDOM-${ami[0]}"
```

Comment:
I've also seen this error of duplicate placement groups. But I'm thinking it might be caused by stale placement groups. It's also possible we can run into dupe placement groups when two jobs are running at the same time and it coincidentally selects the same value.